src/drivers/mpu6000/mpu6000.cpp

```diff
@@ -134,14 +134,22 @@
 #define BITS_FS_1000DPS			0x10
 #define BITS_FS_2000DPS			0x18
 #define BITS_FS_MASK			0x18
-#define BITS_DLPF_CFG_256HZ_NOLPF2	0x00
-#define BITS_DLPF_CFG_188HZ		0x01
-#define BITS_DLPF_CFG_98HZ		0x02
-#define BITS_DLPF_CFG_42HZ		0x03
-#define BITS_DLPF_CFG_20HZ		0x04
-#define BITS_DLPF_CFG_10HZ		0x05
-#define BITS_DLPF_CFG_5HZ		0x06
-#define BITS_DLPF_CFG_2100HZ_NOLPF	0x07
+#define BITS_GYRO_DLPF_CFG_256HZ_NOLPF2	0x00
+#define BITS_GYRO_DLPF_CFG_188HZ	0x01
+#define BITS_GYRO_DLPF_CFG_98HZ		0x02
+#define BITS_GYRO_DLPF_CFG_42HZ		0x03
+#define BITS_GYRO_DLPF_CFG_20HZ		0x04
+#define BITS_GYRO_DLPF_CFG_10HZ		0x05
+#define BITS_GYRO_DLPF_CFG_5HZ		0x06
+#define BITS_GYRO_DLPF_CFG_2100HZ_NOLPF	0x07
+#define BITS_ACC_DLPF_CFG_1046HZ_NOLPF	0x00
+#define BITS_ACC_DLPF_CFG_218HZ		0x01
+#define BITS_ACC_DLPF_CFG_99HZ		0x02
+#define BITS_ACC_DLPF_CFG_44HZ		0x03
+#define BITS_ACC_DLPF_CFG_21HZ		0x04
+#define BITS_ACC_DLPF_CFG_10HZ		0x05
+#define BITS_ACC_DLPF_CFG_5HZ		0x06
+#define BITS_ACC_DLPF_CFG_420		0x07
```

Comment:
@davids5 I made this mods to include the reg options for the 20608 DLPF regs only. There are more where the values are different for both the MPU and IMC series. Since we are keeping the drivers together for maintenance I'll make more for the other values that don't agree with each other.

What would you say is a good naming convention for this static variables? 
BITS_MPU_GYRO_DLPF_CFG
BITS_ICM_GYRO_DLPF_CFG good?